[2024-04-19 02:19:48,352] ERROR log_exception() L838  Exception on /consulta [GET] - call_trace=C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\flask\app.py L838 
Traceback (most recent call last):
  File "C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\flask\app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\flask\app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\flask_cors\extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\flask\app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\flask\app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\flask_openapi3\scaffold.py", line 117, in view_func
    response = func(**func_kwargs)
               ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\app.py", line 106, in get_consulta
    id_da_consulta = query.id
                     ^^^^^^^^
  File "C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\pydantic\main.py", line 792, in __getattr__
    raise AttributeError(f'{type(self).__name__!r} object has no attribute {item!r}')
AttributeError: 'ConsultaBuscaSchema' object has no attribute 'id'
[2024-04-19 02:19:48,365] INFO _log() L96   127.0.0.1 - - [19/Apr/2024 02:19:48] "[35m[1mGET /consulta HTTP/1.1[0m" 500 - - call_trace=C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-19 02:36:31,463] INFO _log() L96   127.0.0.1 - - [19/Apr/2024 02:36:31] "GET /consultas HTTP/1.1" 200 - - call_trace=C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-19 02:37:24,692] INFO _log() L96   127.0.0.1 - - [19/Apr/2024 02:37:24] "POST /consulta HTTP/1.1" 200 - - call_trace=C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-19 02:37:37,025] INFO _log() L96   127.0.0.1 - - [19/Apr/2024 02:37:37] "OPTIONS /consulta?paciente=dr%20Leandro HTTP/1.1" 200 - - call_trace=C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-19 02:37:37,061] INFO _log() L96   127.0.0.1 - - [19/Apr/2024 02:37:37] "DELETE /consulta?paciente=dr%20Leandro HTTP/1.1" 200 - - call_trace=C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-19 02:37:50,878] INFO _log() L96   127.0.0.1 - - [19/Apr/2024 02:37:50] "OPTIONS /consulta?paciente=fvf HTTP/1.1" 200 - - call_trace=C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-19 02:37:50,917] INFO _log() L96   127.0.0.1 - - [19/Apr/2024 02:37:50] "DELETE /consulta?paciente=fvf HTTP/1.1" 200 - - call_trace=C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-19 13:52:24,379] INFO _log() L96   127.0.0.1 - - [19/Apr/2024 13:52:24] "GET /openapi/ HTTP/1.1" 200 - - call_trace=C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-19 13:52:24,452] INFO _log() L96   127.0.0.1 - - [19/Apr/2024 13:52:24] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-19 13:52:24,452] INFO _log() L96   127.0.0.1 - - [19/Apr/2024 13:52:24] "[36mGET /openapi/static/images/redoc.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-19 13:52:24,463] INFO _log() L96   127.0.0.1 - - [19/Apr/2024 13:52:24] "[36mGET /openapi/static/images/rapidoc.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\werkzeug\_internal.py L96  
